packages/nuxt/src/module.ts

```diff
@@ -97,7 +94,11 @@ const module: NuxtModule<ModuleOptions> = defineNuxtModule<ModuleOptions>({
     }
 
     if (options.storesDirs) {
-      for (const storeDir of options.storesDirs) {
+      for (const storeDir of [
+        ...options.storesDirs,
+        /* @ts-expect-error storesDirs isn't on base NuxtOptions type */
+        ...(nuxt.options.pinia?.storesDirs || []),
```

Comment:
This doesn't seem necessary

Reply:
I think this was added so that apps that extend layers using this module can still provide `storesDirs` that would be auto-imported. For example, if myapp had stores in a `mystores` directory, it could add those, while still importing stores the layer defines in its own context.

Reply:
If the layer adds options, I suppose they should appear in the `options` object. If not, it might be an intentional design for layers in Nuxt. Or are you referring to something else?

Reply:
Yes, thank you for correcting. I added tests and used explicit `storesDirs` in the layer config now to ensure that its stores get added.